int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -418,39 +431,45 @@ function edqSetGlobalIntuitiveConfiguration() {
 	];
 }
 /**
-* Pro Web - Address (Verification Engine)
-* Sets the configuration to use Pro Web - Address (Verification Engine)
-*/
+ * Pro Web Address Verification callback
+ */
 function edqValidateAddressCallBack() {
 	var edqProWebResponse = document.querySelector("#form-submit");
 	if (edqProWebResponse.hasAttribute("edq-metadata")) {
 		var edqProWebMetaDataJSON = JSON.parse(edqProWebResponse.getAttribute("edq-metadata"));
 		document.querySelector("#form-submit").style.display = "none";
-		edqStateLineSelector.value = edqProWebMetaDataJSON["State code"];
+		edqStateLineSelector.value = edqProWebMetaDataJSON["address.province"];
```

Comment:
Are you sure this is the right key to be using? It looks like a Global Intuitive key.

Reply:
Yes, on the new layout (Default) the Elements are going to be the same as Global Intuitive.